de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/fskdbview/FSKDBViewNodeModel.java

```diff
@@ -18,13 +18,11 @@
  */
 package de.bund.bfr.knime.fsklab.v1_9.fskdbview;
 
-import java.io.File;
```

Comment:
This file is changed in https://github.com/SiLeBAT/FSK-Lab/pull/680. Once that PR is merged you could merge development into this branch to bring the changes.

Reply:
Sure, waiting for PR merging :)